todo/src/actions.js

```diff
@@ -0,0 +1,46 @@
+
+export function addTodo(todo,important) {
+    // console.log('addTodo action maker has been triggered ', todo);
+    return {
+        type: "ADD_TODO",
```

Comment:
While it's not required to create the ACTION_TYPES as variables instead of strings , you may want to start doing it as it is a regular practice in the industry to declare  your strings that will be used in more than one place as a constant variable in your application. It is common to see a file of just constants in projects